testsuite/tests/libratest/smoke_test.rs

```diff
@@ -930,6 +930,72 @@ fn test_e2e_reconfiguration() {
     );
 }
 
+#[test]
+fn test_e2e_modify_publishing_option() {
+    let (mut env, mut client_proxy) = setup_swarm_and_client_proxy(1, 0);
+    client_proxy.create_next_account(false).unwrap();
+
+    client_proxy
+        .mint_coins(&["mintb", "0", "10"], true)
+        .unwrap();
+    assert_eq!(
+        Decimal::from_f64(10.0),
+        Decimal::from_str(&client_proxy.get_balance(&["b", "0"]).unwrap()).ok()
+    );
+    let script_path = workspace_builder::workspace_root()
+        .join("testsuite/tests/libratest/dev_modules/test_script.mvir");
+    let unwrapped_script_path = script_path.to_str().unwrap();
+    let script_params = &["execute", "0", unwrapped_script_path, "script"];
+    let script_compiled_path = client_proxy.compile_program(script_params).unwrap();
+
+    // Initially publishing option was set to CustomScript, this transaction should be executed.
```

Comment:
Where does this setup happen? Is it the default in `setup_swarm_and_client_proxy`?